src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/TableBuilderController.cs

```diff
@@ -94,22 +94,23 @@ public class TableBuilderController : ControllerBase
             }
 
             return await _tableBuilderService
-                .Query(releaseVersionId, request.AsFullTableQuery(), cancellationToken)
+                .Query(releaseVersionId, request.AsFullTableQuery(), request.BoundaryLevel, cancellationToken)
                 .HandleFailuresOr(Ok);
         }
 
         // Note that releaseVersionId is not necessary for this method to function any more in the Data API, but remains in place
         // both in order to support legacy URLs in bookmarks and in content, but also to remain consistent with the equivalent
         // endpoint in the Admin API, which does require the Release Id in order to differentiate between different 
         // DataBlockVersions rather than simply picking the latest published one.
-        [HttpGet("tablebuilder/release/{releaseVersionId:guid}/data-block/{dataBlockParentId:guid}")]
+        [HttpGet("tablebuilder/release/{releaseVersionId:guid}/data-block/{dataBlockParentId:guid}/{boundaryLevelId:long?}")]
```

Comment:
See similar comment in Admin about changing this to use a query parameter `[FromQuery] long? boundaryLevelId`